src/cbl-mariner/2.0/fpm/amd64/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+RUN tdnf install -y \
+    ruby \
```

Comment:
Could you just indent this 4 spaces since it's a continuation of the same command from the previous line?